components/chunkprogressbar/overview.md

```diff
@@ -33,48 +30,11 @@ To use the Telerik ChunkProgressBar in your Blazor application:
 }
 ````
 
-
->caption Component namespace and reference
-
-````CSHTML
-<TelerikChunkProgressBar Value="@ChunkProgressBarValue" Max="@MaxValue" @ref="@MyChunkProgressBar" />
-
-@code {
-    Telerik.Blazor.Components.TelerikChunkProgressBar MyChunkProgressBar { get; set; }
-
-    public double MaxValue { get; set; } = 10;
-    public double ChunkProgressBarValue { get; set; } = 3;
-}
-````
-
-## Features
-
-The ChunkProgressBar provides the following features:
-
-* `Class` - the CSS class that will be rendered on the main wrapping element. You can use it to cascade styles more easily.
-* `Max` - `double`, defaults to `100` - the maximum value of the ChunkProgressBar. It must be greater than `0`.
-* `Value` - `double` - the value of the ChunkProgressBar. This value indicates the progress of the tracked process and is distributed in the chunks (sections) of the ChunkProgressBar. It is a fraction of the `Max`. Read mode in the [Chunk Count, Value and MaxValue](#chunk-count-value-and-maxvalue) section.
-* `ChunkCount` - `unsigned int`, defaults to `5` - the number of chunks the ChunkProgressBar will be separated into.
-* `Orientation` - you can control the orientation of the Progress Bar, through the `ProgressBarOrientation` enum, with members:
-    * `Horizontal` - this is the default value
-    * `Vertical`
-* `Indeterminate` - `bool`, defaults to `false` - see the [Indeterminate]({%slug chunkprogressbar-indeterminate-state%}) article for more information.
-
 ## Chunk Count, Value and MaxValue
 
-The following examples explain how the three parameters of the component tie together to determine how many chunks are filled in.
+This section explains the connection between the [ChunkCount, Value, and MaxValue parameters](#chunkprogressbar-parameters). In the [Matching Values](#matching-values) and the [Non-matching Values](#non-matching-values) sub-sections you can see how the ChunkProgressBar will render its chunks in these scenarios. 
```

Comment:
"This section" and the linked ones are at the same level, so the latter are not sub-sections. Either change the heading structure, or reword the paragraph. You can also consider removing the "Chunk Count..." heading and merge this intro text with the Basic section - i.e. treat it as "Next Steps" of the Basic section.

Reply:
Actually, the Matching and Non-matching must be h3, so the heading structure must be changed to reflect the intent of the section.